BREAKING.md

```diff
@@ -24,6 +24,7 @@ There are a few steps you can take to write a good change note and avoid needing
 - [Set raiseContainerWarning property as optional parameter on IContainerContext](#Set-raiseContainerWarning-property-as-optional-parameter-on-IContainerContext)
 - [Consolidate fluidErrorCode and message on FF Errors](#Consolidate-fluidErrorCode-and-message-on-FF-Errors)
 - [Doing operations not allowed on deleted sub directory](#Doing-operations-not-allowed-on-deleted-sub-directory)
+ - [Removing Commit from TreeEntry and commits from SnapShotTree](#Removing-Commit-from-TreeEntre-and-commits-from-SnapShotTree)
```

Comment:
can you also create a new section for 0.59 and put this in there?  next should be on 0.59.1000 but that change got lost with all the other version stuff that was happening